packages/core/src/internal/io.js

```diff
@@ -89,6 +89,7 @@ function getFnCallDesc(meth, fn, args) {
   }
 
   let context = null
+  // prettier-ignore
```

Comment:
whats the reason behind this comment?

Reply:
In .lintstagedrc we've configured that staged files will run through `prettier` and `eslint --fix`.

If we run prettier only,  it will add semicolons before `[` and `(`.

![image](https://user-images.githubusercontent.com/5550931/41133499-e1f5c21c-6af9-11e8-9a03-05f786ab092a.png)

These semicolons then are removed by `eslint --fix`.

Reply:
so we need to change our setup? otherwise we have to add that comment manually for all cases

Reply:
I think we can turn off *no-extra-semi* rule in ESLint. Using prettier to remove extra semicolons is enough. 😃